### PR TITLE
Add testcase pbc with nullspace

### PR DESCRIPTION
**Pull request by m-frey**
## Description and Context

Introduces a new framework test for 3D RVE periodic boundary conditions with nullspace projection, and registers it.

### CODE REVIEW

File: tests/input_files/rve3d_periodic_bcs_nullspace.4C.yaml [hunk 1]
```diff
@@ -0,0 +1,840 @@
+TITLE:
+  - "Test periodic boundary conditions for a 3D RVE with nullspace projection"
+  - "Verify rigid-body nullspace removal with Krylov-space projection"
```
Comment (mayrmt) [inline tests/input_files/rve3d_periodic_bcs_nullspace.4C.yaml:3]:
```suggestion
  - "Verify rigid-body mode removal through Krylov projection with nullspace vectors"
```

File: tests/input_files/rve3d_periodic_bcs_nullspace.4C.yaml [hunk 2]
```diff
@@ -0,0 +1,840 @@
...
+SOLVER 1:
+  SOLVER: "UMFPACK"
+  NAME: "Structure_Solver"
+STRUCTURAL DYNAMIC:
+  INT_STRATEGY: "Standard"
+  DYNAMICTYPE: "Statics"
+  TIMESTEP: 0.5
+  NUMSTEP: 2
+  MAXTIME: 1
+  M_DAMP: 0.01
```
Comment (mayrmt) [inline tests/input_files/rve3d_periodic_bcs_nullspace.4C.yaml:22]: Time integration is statics, so this is not needed:
```suggestion
```

**Comment by maxfirmbach** [comment]
Fails due to
```terminal
{"DESCRIPTION":0.95,"NAME":"Faa","TYPE":"expression","VARIABLE":0} is not valid under any of the schemas listed in the 'oneOf' keyword

Failed validating "oneOf" in schema["patternProperties"]["^FUNCT[1-9][0-9]*$"]["items"]

On instance["FUNCT1"][1]:
    {"DESCRIPTION":0.95,"NAME":"Faa","TYPE":"expression","VARIABLE":0}
```

**Review by mayrmt — CHANGES_REQUESTED** [review]
Please update the documentation of the test case.

Commits can be squashed.

File: tests/input_files/rve3d_periodic_bcs_nullspace.4C.yaml (hunk 1 above)
Comment (m-frey) [reply tests/input_files/rve3d_periodic_bcs_nullspace.4C.yaml]: ✔️

File: tests/input_files/rve3d_periodic_bcs_nullspace.4C.yaml (hunk 2 above)
Comment (m-frey) [reply tests/input_files/rve3d_periodic_bcs_nullspace.4C.yaml]: ✔️

**Review by maxfirmbach — APPROVED** [review]
Thank you for adding the test case 👍.

**Review by knarfnitram — APPROVED** [review]

**Review by mayrmt — APPROVED** [review]
@m-frey, thanks! Looks good.
